public/app/project-setup/theme/variables/_layout.scss

```diff
@@ -1 +1,2 @@
-$global-padding: 50px;
+$basic-padding: 50px;
```

Comment:
Because padding and column width are no related all the times.
This is an occasional link